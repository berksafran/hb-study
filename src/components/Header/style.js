import styled from "styled-components";

const HeaderContainer = styled.div`
  display: flex;
  flex: 1;
  flex-direction: "row";
  justify-content: space-between;
  align-items: center;
  border-width: 0 0 1px 0;
  border-style: solid;
  padding-bottom: 2px;
  border-color: gray;

  @media only screen and (max-width: 850px) {
    display: flex;
    justify-content: center;
    position: initial;
    flex-wrap: wrap;

    .logo {
      padding-bottom: 15px;
    }
  }

  .logoÂ  {
    mix-blend-mode: luminosity;
  }

  p {
    font-size: 1.4rem;
    font-weight: normal;
    margin: 0;
  }

  strong {
    font-weight: 800;
  }

  .vote {
    font-weight: 200;
  }
`;

export { HeaderContainer };
