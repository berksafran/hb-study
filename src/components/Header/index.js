import React from "react";
import { Image } from "react-bootstrap";
import { HeaderContainer } from "./style";

function Header() {
  return (
    <HeaderContainer>
      <Image
        width={250}
        src="assets/hepsiburada_logo.jpg"
        alt="logo"
        className="logo"
      />
      <p>
        <strong>Link</strong>
        <span className="vote">VOTE</span> Challenge
      </p>
    </HeaderContainer>
  );
}

export default Header;
