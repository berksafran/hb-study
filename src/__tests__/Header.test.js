import React from 'react'
import renderer from 'react-test-renderer'
import Header from '../components/Header';

test('Header component works', () => {
  const header_tree = renderer.create(<Header />).toJSON()
  expect(header_tree).toMatchSnapshot()
})