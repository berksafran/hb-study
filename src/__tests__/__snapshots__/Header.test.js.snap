// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component works 1`] = `
<div
  className="sc-bdVaJa hekzgj"
>
  <img
    alt="logo"
    className="logo"
    src="assets/hepsiburada_logo.jpg"
    width={250}
  />
  <p>
    <strong>
      Link
    </strong>
    <span
      className="vote"
    >
      VOTE
    </span>
     Challenge
  </p>
</div>
`;
